cinema/views.py

```diff
@@ -62,3 +87,57 @@ def get_serializer_class(self):
             return MovieSessionDetailSerializer
 
         return MovieSessionSerializer
+
+    def get_queryset(self):
+        date = self.request.query_params.get("date")
+        movies_id = self.request.query_params.get("movie")
+        self.queryset = self.queryset.select_related("cinema_hall", "movie")
+
+        if date:
+            date = date.split("-")
+            self.queryset = self.queryset.filter(show_time__year=date[0],
+                                                 show_time__month=date[1],
+                                                 show_time__day=date[2])
```

Comment:
Don't use split for date format. You can try .filter(show_time=date), you have already date the relevant format.